src/app/features/benefits-bundle/benefits-training-discounts/benefits-training-discounts.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { JourneyType } from '@core/breadcrumb/breadcrumb.model';
+import { BreadcrumbService } from '@core/services/breadcrumb.service';
+import { EstablishmentService } from '@core/services/establishment.service';
+
+@Component({
+  selector: 'app-benefits-training-discounts',
+  templateUrl: './benefits-training-discounts.component.html',
+})
+export class BenefitsTrainingDiscountsComponent implements OnInit {
+  public pages;
```

Comment:
```suggestion
  public pages: Page;
```
Could you use the `Page` model here?